Data/Help/1/scriptarray.html

```diff
@@ -0,0 +1,42 @@
+<html>
+<head>
+    <title>Help for Deadly Rooms of Death</title>
+</head>
+<body bgcolor="#FFFFFF">
+<img src="images/drodlogo.jpg" />
+<hr />
+<h3><u>Using Arrays in Scripts</u></h3>
+    <p>Script variables can be configured as arrays, by starting the variable name with a hash (#) or an at symbol (@). 
+    Variables starting with a hash a global arrays, while variables starting with an at symbol are local to individual characters. 
+    Once a variable is created, you cannot change if it is an array variable (but its locality can be changed). Array variables can only store integer values, not text.</p>
+    <p>An array variable is effectively a list of values, each associated with a specific index. Therefore, an arbitary number of values can be stored under a single variable name. 
```

Comment:
"a list of values, each associated with a specific index" --> "a set of index-value pairs, where each index has a value (defaulting to 0)"